### PR TITLE
Fix projects test suite

### PR DESCRIPTION
**Pull request by shailysangwan**
Please consider merging only after https://github.com/Cadasta/cadasta-test/issues/21 is solved.
Tested using `webdriver.maximize_window()`, and all tests were passing

Issue https://github.com/Cadasta/cadasta-test/issues/22 possibly refers to some of the bugs fixed through this.

### CODE REVIEW

**Comment by shailysangwan** [comment]
@manoramahp could you also review this please?

**Comment by linzjax** [comment]
Could you be more specific about what problem this is solving and how your solution fixes it?

File: selenium_tests/projects/location_relationships.py [hunk 1]
```diff
@@ -20,28 +20,17 @@ def test_attach_relationship_to_new_location(self):
...
         self.wd.wait_for_xpath("//h2[contains(text(), 'Project Overview')]")
         self.wd.find_element_by_link_text("Add location").click()
-
-        self.wd.wait_for_xpath("//h3[contains(text(), 'Draw location on map')]")
-        page_state = self.wd.execute_script('return document.readyState;')
-        while page_state != 'complete':
-            page_state = self.wd.execute_script('return document.readyState;')
-
-        self.wd.find_element_by_css_selector("a.leaflet-draw-draw-marker").click()
+        self.wd.wait_for_css("a.leaflet-draw-draw-marker").click()
```
Comment (shailysangwan) [inline selenium_tests/projects/location_relationships.py]: The purpose of the lines deleted is to wait until the draw-marker button is visible.
They can be replaced by a one liner.

**Comment by shailysangwan** [comment]
@linzjax I've closed the PR for now.
It was meant to fix the tests that are currently commented out in the Projects suite because of the existence of bugs in their implementations. While fixing them, I noticed that tests that do work fine can further be improved.
So the changes included that as well.
I plan to fix them in accordance with the timeline that I have decided in my Outreachy application.
Can you please take a look at it?

**Pull request closed without merging**